src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.SectionExceptionCode;
+import nextstep.subway.utils.NumberUtil;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+
+    @Column(nullable = false)
+    private int distance;
+
+    protected Distance() {
+    }
+
+    public Distance(int distance) {
+        validateDistance(distance);
+        this.distance = distance;
+    }
+
+    private void validateDistance(int distance) {
+        if(NumberUtil.isNotPositiveNumber(distance)) {
+            throw new IllegalArgumentException(
+                    SectionExceptionCode.DO_NOT_ALLOW_NEGATIVE_NUMBER_DISTANCE.getMessage());
+        }
+    }
+
+    public Distance minus(int distance) {
+        if(this.distance <= distance) {
+            throw new IllegalArgumentException(SectionExceptionCode.INVALID_DISTANCE.getMessage());
+        }
+
+        return new Distance(this.distance - distance);
+    }
+
+    public Distance plus(int distance) {
+        return new Distance(this.distance + distance);
+    }
```

Comment:
```suggestion
    public Distance minus(Distance target) {
        if(this.distance <= target.distance) {
            throw new IllegalArgumentException(SectionExceptionCode.INVALID_DISTANCE.getMessage());
        }

        return new Distance(this.distance - target.distance);
    }

    public Distance plus(Distance target) {
        return new Distance(this.distance + target.distance);
    }
```
`Distance`를 정의한김에 `Distance`끼리 연산하게하는건 어떨까요? 😀